python/ray/serve/api.py

```diff
@@ -222,7 +222,8 @@ def create_backend(
         Args:
             backend_tag (str): a unique tag assign to identify this backend.
             func_or_class (callable, class): a function or a class implementing
-                __call__.
+                __call__, returning a JSON-serializable object or a
+                serializable Starlette Response object.
```

Comment:
All Starlette response objects should be serializable, right? If so, can drop the qualifier - it'll probably confuse people